### PR TITLE
SOC-1204 As a user on desktop I want that the new auth modal replaces the current forced login modal

### PR DESCRIPTION
**Pull request by rafalkalinski**
https://wikia-inc.atlassian.net/browse/SOC-1204

### CODE REVIEW

File: front/scripts/auth/common/AuthUtils.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class AuthUtils {
+
+	public static authSuccessCallback (url?: string): void {
+		if (window.parent) {
+			window.parent.postMessage({isUserAuthorized: true}, location.protocol + "//" + location.hostname);
+			return;
+		} else if (url) {
+			window.location.href = url;
+			return;
```
Comment (bkoval) [inline front/scripts/auth/common/AuthUtils.ts]: No need for a return statement here, change to window.location will reload the page.

File: front/scripts/auth/common/AuthUtils.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	}
+
+	public static loadUrl (url?: string): void {
+		var win: Window;
+
+		win = (pageParams.isModal ? window.parent : window);
+
+		if (url) {
+			win.location.href = url;
+			return
```
Comment (bkoval) [inline front/scripts/auth/common/AuthUtils.ts]: No return statement needed here as well.

File: front/scripts/auth/common/AuthUtils.ts (hunk 1 above)
Comment (rafalkalinski) [inline front/scripts/auth/common/AuthUtils.ts]: I'm used to left return statement after location.href.

http://blog.johnryding.com/post/1548525629/windowlocationhref-and-executing-code-placed-after

File: front/scripts/auth/common/AuthUtils.ts (hunk 2 above)
Comment (rafalkalinski) [inline front/scripts/auth/common/AuthUtils.ts]: I'm used to left return statement after location.href.

http://blog.johnryding.com/post/1548525629/windowlocationhref-and-executing-code-placed-after

File: front/scripts/auth/common/AuthUtils.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class AuthUtils {
+
+	public static authSuccessCallback (url?: string): void {
+		if (window.parent) {
+			window.parent.postMessage({isUserAuthorized: true}, location.protocol + "//" + location.hostname);
```
Comment (kvas-damian) [inline front/scripts/auth/common/AuthUtils.ts]: Are we sure protocol will match on Auth pages vs regular pages?

**Comment by bkoval** [comment]
LGTM :bikini:

**Comment by kvas-damian** [comment]
LGTM :jack_o_lantern:

**Pull request merged**